src/components/ConfirmItemArrival/ConfirmItemArrival.test.js

```diff
@@ -35,7 +35,7 @@ describe('ConfirmItemArrival', () => {
     expect(screen.getByText(labelIds.paneTitle)).toBeVisible();
   });
 
-  it('should trigger NavigationMenu with correct props', () => {
+  it('should render navigation menu with correct props', () => {
```

Comment:
I think trigger was better because in the test you check component call, not component render.

Reply:
Fixed